Game Here/project.godot

```diff
@@ -11,9 +11,9 @@ config_version=5
 [application]
 
 config/name="Video Game Design"
-run/main_scene="uid://c7sp4y02erkyp"
-config/features=PackedStringArray("4.4", "Forward Plus")
-config/icon="uid://d2cj4uugc0uhp"
+run/main_scene="res://obby.tscn"
+config/features=PackedStringArray("4.3", "Forward Plus")
```

Comment:
And you wonder why it doesn't work?
And then you go and merge despite the big red X?
And then you complain about it?

I'm sorry, that's harsh. But mostly true.

Reply:
Fixed by 0c7ffd59c92ec70dfdff6057cfb66b2de5754dc5. Hopefully it works.